code-samples/eventing/bookstore-sample-app/ML-sentiment-analysis/README.md

```diff
@@ -0,0 +1,413 @@
+## What Knative features will we learn about?
+
+- The easiness to use **Knative Function** to deploy your service, and make it be managed by **Knative Serving**, which give you the ability to auto-scale your service to zero, and scale up to handle the demand.
+
+## What does the final deliverable look like?
+A running serverless Knative function that contains a python application that can receives the new review comments as CloudEvent and returns the sentiment classification of the input text as CloudEvent.
+
+The function's output will be only from 
+- Positive
+- Neutral
+- Negative
+## Install Prerequisites
+
+---
+(Warning box: please make sure you have a running cluster with Knative Eventing and Serving installed. If not, click here. And you have the container registry ready.)
+
+---
+
+### Prerequisite 1: Install Knative `func` CLI
+Knative Function enables you to easily create, build, and deploy stateless, event-driven functions as [Knative Services](https://knative.dev/docs/serving/services/#:~:text=Knative%20Services%20are%20used%20to,the%20Service%20to%20be%20configured) by using the func CLI.
+
+In order to do so, you need to install the `func` CLI.
+You can follow the [official documentation](https://knative.dev/docs/getting-started/install-func/) to install the `func` CLI.
+
+Running `func version` in your terminal to verify the installation, and you should see the version of the `func` CLI you installed.
+
+## Implementation
+The process is straightforward:
+- Begin by utilizing the `func create` command to generate your code template.
+- Next, incorporate your unique code into this template.
+- Finally, execute `func deploy` to deploy your application seamlessly to the Kubernetes cluster.
+
+This workflow ensures a smooth transition from development to deployment within the Knative Functions ecosystem.
+
+---
+a warning box:
+(Troubleshooting: if you see `command not found`, you may need to add the `func` CLI to your PATH.)
+---
+### Step 1: Create a Knative Function template
+
+Create a new function using the `func` CLI:
+
+```bash
+func create -l <language> <function-name>
+```
+
+In this case, we are creating a python function, so the command will be:
+   
+```bash
+func create -l python sentiment-analysis
+```
+
+This command will create a new directory with the name `sentiment-analysis` and a bunch of files in it. The `func` CLI will generate a basic function template for you to start with.
+
+You can find all the supported languages templates [here](https://knative.dev/docs/functions/).
+
+The file tree will look like this:
+```bash
+sentiment-analysis
+├── func.yaml                 
+├── .funcignore               
+├── .gitignore                
+├── requirements.txt         
+├── app.sh
+├── func.yaml
```

Comment:
Duplicate with L63?

Reply:
Good catch! Thanks for pointing that out! @tico88612